docs/STRUCTURE.md

```
@@ -0,0 +1,207 @@
+- ✅ Modern patterns
+- ✅ Well documented
+
+## 🚀 Adding New Service
+
+1. **Create chart** (if custom):
+   ```bash
+   helm create charts/myservice
+   ```
+
+2. **Add environment values**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)